libavpipe/src/avpipe_xc.c

```diff
@@ -4395,6 +4426,13 @@ check_params(
         params->n_audio = 1;
     }
 
+    for (int i=0; i<params->n_audio; i++) {
```

Comment:
I thought when we discussed things earlier, it was reasonable for multiple audios to share the same index if the operation wanted multiple copies of the same audio track? Is this safe for the `AudioPan` or `AudioMerge` usecases? Just want to double check.

Reply:
No, as I discussed we want to have audio join/merge to use the same audio index. This is not implemented yet, and we can open up this check when we (I) implement this new feature.
Right now, I just avoid having multiple audio so nobody makes a mistake in setting audio indexes.

Reply:
Ah, that makes perfect sense, yes.